### ISSUE TITLE
unintended behavior with marketing campaign tracking in the minified version

### ISSUE THREAD

**Issue by dohyungp** [mixpanel/mixpanel-js#403]
Hello, Mixpanel Team. 

Although the SDK was modified to make marketing campaigns un-persistent in version 2.48.0, there is still an issue in the minified version where UTM parameters are being stored persistently.

I compared the minified version with the original code and identified that the issue is caused by minifying the JSON key.

v2.48.0, https://cdn.mxpnl.com/libs/mixpanel-2-latest.min.js
```js
e.prototype.mb=function(){this.persistence.hc(u.referrer);this.c("store_google")&&
this.m(c.info.ka(),{xd:D})
```

v2.48.0, https://cdn.mxpnl.com/libs/mixpanel-2-latest.js
```js
MixpanelLib.prototype._set_default_superprops = function() {
        this['persistence'].update_search_keyword(document$1.referrer);
        if (this.get_config('store_google')) {
            this.register(_.info.campaignParams(), {persistent: false});
        }
        if (this.get_config('save_referrer')) {
            this['persistence'].update_referrer_info(document$1.referrer);
        }
    };
```

Warm Regards

**Comment by tdumitrescu** [comment]
Hi @dohyungp, there wasn't any intended change to attribution code in v2.48.0; see release notes here: https://github.com/mixpanel/mixpanel-js/releases/tag/v2.48.0. The team working on attribution has prepared a fix for the minification bug you mention, which will be released soon (probably as v2.48.1).

**Comment by tdumitrescu** [comment]
Release [2.48.1](https://github.com/mixpanel/mixpanel-js/releases/tag/v2.48.1) is out now, which consolidates behavior across minified and unminified versions (the current behavior is always to persist the params).
